CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+
+Avoid using offensive or harassing package names, nicknames, or other identifiers that might detract from a friendly, safe, and welcoming environment for all. Harassment includes, but is not limited to: harmful or prejudicial verbal or written comments related to gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics; inappropriate use of nudity, sexual images, and/or sexually explicit language in public spaces; threats of physical or non-physical harm; deliberate intimidation, stalking or following; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact; and unwelcome sexual attention.
+
+# Reporting Violations of this Code of Conduct
+If you believe someone is harassing you or has otherwise violated this Code of Conduct, please contact any of the administrators listed at the top of this document and send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
+
+# Consequences
+All content published to the Passage GitHub Community, including user account credentials, is hosted at the sole discretion of the 1Password Developer community administrators.
+
+Unacceptable behavior from any community member, including employees, customers, or others with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the 1Password Developer community administrators may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
+
+# Addressing Grievances
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, please notify the 1Password Developers Slack Community admin team either by posting in [`#moderators`](https://1password-devs.slack.com/archives/C0331DJL882) or reaching out privately via DM to any of the administrators listed at the top of this document.
+
+We will do our best to ensure that your grievance is handled appropriately.
+
+In general, we will choose the course of action that we judge as being most in the interest of fostering a safe and friendly community.
+
+# Contact Information
+Please contact any of the administrators listed at the top of this document if you need to report a problem or address a grievance related to an abuse report.
+
+You're encouraged to contact the admin team in cases where you're unsure if something is or isn't appropriate content. We are happy to provide guidance to help you be a successful part of our community.
+
+# Learn More
+Learn more at the [1Password Developer Community](https://developer.1password.com/code-of-conduct/).
```

Comment:
Maybe just remove the Learn More section?

Reply:
The acceptance criteria included specifically linking to this URL. This was by best attempt at finding an organic way of doing so

Reply:
Got ya. Yeah, this is slightly confusing. They aren't necessarily learning more there, it is just another version of this. Maybe this is helpful? I think we need to lean on the 1Password influence to explain why we are sending them back there. Thoughts?

```suggestion
Passage by 1Password has more resources at our larger [1Password Developer Community](https://developer.1password.com/code-of-conduct/).
```

Reply:
I do like the framing of this sentence better